scripts/makeLegacyPlots.py

```diff
@@ -497,7 +529,6 @@ def scaleStatSystUncertaintyBandForStack (grUncert,bkgSum):
     parser.add_argument('--sel', dest='sel', help='selection name', default=None)
     parser.add_argument('--name', dest='name', help='selection name for plot', default=None)
     parser.add_argument('--dir', dest='dir', help='analysis output folder name', default="./")
-    parser.add_argument('--tag', dest='tag', help='plots output folder name', default="./")
```

Comment:
You removed the wrong one. Practically it does not change, but the description (`help`) of this parameter is different.

Reply:
ah ok I see that you removed the one that was commented, my bad I didn't notice before. In any case this one should be kept and the other one removed.

Reply:
I added one more variable in the sh file called "input" which goes in the "dir" name and I changed the description of the "tag" variable which is now appended to the output folder name and can be whatever one wants.